app/controllers/extraction_definitions_controller.rb

```diff
@@ -89,6 +102,15 @@ def update_redirect_path
     end
   end
 
+  def successful_clone_path(clone)
```

Comment:
This method should return a path so that it follows its name. You'll also be able to do 

`redirect_to successful_clone_path(clone), notice: t('.success')` in one line :)

Reply:
Nice! Thanks @paul-mesnilgrente